src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -218,6 +219,19 @@
             <argument type="service" id="api_platform.property_accessor" />
         </service>
 
+        <service id="api_platform.identifier_normalizer" class="ApiPlatform\Core\Identifier\IdentifierNormalizer" public="false">
+            <argument type="service" id="api_platform.metadata.property.name_collection_factory" />
+            <argument type="service" id="api_platform.metadata.property.metadata_factory" />
+            <argument type="service" id="api_platform.identifier.composite_identifier_normalizer" />
+            <argument type="collection" /> <!-- normalizers -->
+        </service>
+
+        <service id="api_platform.identifier.composite_identifier_normalizer" class="ApiPlatform\Core\Identifier\CompositeIdentifierNormalizer" public="false" />
+
+        <service id="api_platform.identifier.date_normalizer" class="Symfony\Component\Serializer\Normalizer\DateTimeNormalizer" public="false">
+            <tag name="api_platform.identifier_normalizer" />
+        </service>
```

Comment:
I'm not sure that this is allowed, lmk if there's something better to use instead.

Reply:
What about adding uuid normalizer? It is very common I think.

Reply:
added below :) Just need to test it!

Reply:
Why would not it be?